server/app.py

```diff
@@ -1455,12 +1492,14 @@ def log_rdo(type=None, contact=None, account=None, subscription=None):
     rdo.encouraged_by = sub_meta.get("encouraged_by", None)
     rdo.lead_source = "Stripe"
     rdo.amount = sub_meta.get("donor_selected_amount", 0)
-    rdo.installments = None
     rdo.installment_period = installment_period
-    rdo.open_ended_status = "Open"
+    rdo.open_ended_status = donation_type_info.get("open_ended_status", None)
```

Comment:
**question:** For circle, you use "None" as a string. Is that the intended default or is the intention to not set it at all for the default fallback

Reply:
a "None" string is the intended default... not setting it at all seems to cause issues with the RDO properly creating child Opportunities

Reply:
as seen with the business membership donation